ldaptor/test/test_ldiftree.py

```diff
@@ -21,6 +23,18 @@ def writeFile(path, content):
     f.close()
 
 
+skipIfWindowsOrRoot = stdlib_unittest.skipIf(
+    sys.platform == "win32" or os.getuid() == 0,
+    "Can't test on windows or as root",
+)
+
+
+skipIfWindows = stdlib_unittest.skipIf(
+    sys.platform == "win32",
+    "TODO: fails on windows",
```

Comment:
I think that this needs a dedicated issue and the TODO: can include the link to that issue.